test/e2e/framework/test_context.go

```diff
@@ -526,72 +522,23 @@ func AfterReadingAllFlags(t *TestContextType) {
 	}
 
 	if TestContext.ReportDir != "" {
-		ginkgo.ReportAfterSuite("Kubernetes e2e JUnit report", writeJUnitReport)
+		ginkgo.ReportAfterSuite("Kubernetes e2e JUnit report", func(report ginkgo.Report) {
+			// With Ginkgo v1, we used to write one file per
+			// parallel node. Now Ginkgo v2 automatically merges
+			// all results into a report for us. The 01 suffix is
+			// kept in case that users expect files to be called
+			// "junit_<prefix><number>.xml".
+			junitReport := path.Join(TestContext.ReportDir, "junit_"+TestContext.ReportPrefix+"01.xml")
+
+			// writeJUnitReport generates a JUnit file in the e2e
+			// report directory that is shorter than the one
+			// normally written by `ginkgo --junit-report`. This is
+			// needed because the full report can become too large
+			// for tools like Spyglass
+			// (https://github.com/kubernetes/kubernetes/issues/111510).
+			//
+			// Users who want the full report can use `--junit-report`.
```

Comment:
those comments seems like dup with https://github.com/kubernetes/kubernetes/pull/113538/files#diff-daa6e9e53080eff01e28fece2f47c94a37d88d98717c54a0e8ab3d488d1aaaa4R25-R28 and could be removed from here, btw, I am fine with it.

Reply:
Those are two different items that get documented. I think it's better to repeat some information than to assume that the reader remembers a comment that is made elsewhere.